Marlin/src/lcd/menu/menu_tune.cpp

```diff
@@ -129,6 +133,10 @@ void menu_tune() {
     #endif // HOTENDS > 2
   #endif // HOTENDS > 1
 
+  #if ENABLED(SINGLENOZZLE)
+    MENU_MULTIPLIER_ITEM_EDIT(uint16, MSG_NOZZLE MSG_STANDBY, &singlenozzle_temp[(int)!active_extruder], 0, HEATER_0_MAXTEMP - 15);
```

Comment:
Please don't combine language strings like this. Different languages use different word-orders.